pkg/attestlib/pgp.go

```diff
@@ -82,6 +84,13 @@ func NewPgpSigner(privateKey []byte) (Signer, error) {
 		return nil, fmt.Errorf("expected 1 key in keyring, got %d", len(keyring))
 	}
 	key := keyring[0]
+
+	if passphrase != "" {
+		err := key.PrivateKey.Decrypt([]byte(passphrase))
+		if err != nil {
+			return nil, errors.Wrap(err, "could not decrypt private key")
+		}
+	}
```

Comment:
If passphrase is "" but the privetekey is passphrase-protected, is there a way to know it due signer creation time?

Reply:
Yes, added this check.